libs/sdk-ui-pivot/src/impl/data/tests/loadingRenderer.test.tsx

```diff
@@ -39,33 +39,37 @@ describe("RowLoadingElement", () => {
         expect(wrapper.html()).toEqual('<span class="s-value s-loading-done">3.14</span>');
     });
 
-    describe("'LoadingComponent' color property", async () => {
-        const LoadingRenderer = await createTestRenderer();
+    describe("'LoadingComponent' color property", () => {
```

Comment:
out of curiosity... what is wrong with the old version? i did it like this and thought it's ok to do